testing/integration-tests/src/full_client/client/unstable_rpcs.rs

```diff
@@ -122,11 +126,12 @@ async fn chainhead_unstable_storage() {
 
     let mut blocks = rpc.chainhead_unstable_follow(false).await.unwrap();
     let event = blocks.next().await.unwrap().unwrap();
-    let hash = match event {
-        FollowEvent::Initialized(init) => init.finalized_block_hash,
+    let hashes = match event {
+        FollowEvent::Initialized(init) => init.finalized_block_hashes,
```

Comment:
Nit: Personally for these tests I'd just do something like:

```rust
let hash = match event {
    FollowEvent::Initialized(init) => init.finalized_block_hashes.last().unwrap(),
    _ => panic!("Unexpected event"),
};
```

The spec guarantees we'll get at least 1 anyway